### ISSUE TITLE
Exception cause deeplearning model to fail when calling fit with TimeSeriesPredictor

### ISSUE THREAD

**Issue by BrunoPinos** [autogluon/autogluon#3612]
Warning: Exception caused DeepAR to fail during training... Skipping this model.
	
 you tried to log -1 which is currently not supported. Try a dict or a scalar/tensor.

![image](https://github.com/autogluon/autogluon/assets/58994038/bc2b1370-3283-4c9c-9739-f8caddf960ef)
![image](https://github.com/autogluon/autogluon/assets/58994038/e0e735a6-a743-4b4e-b7de-235514dab6f0)

gluonts == 0.13.7
autogluon.timeseries == 0.8.1

**Comment by shchur** [comment]
Can you please check the versions of `pytorch_lightning` and `lightning` in your environment (if present)?

I wasn't able to reproduce this behavior with a fresh install using `pip install autogluon==0.8.1`. I suspect that you have `pytorch_lightning >= 2.0`, which is incompatible with `autogluon==0.8.*` (this AutoGluon version expects `pytorch_lightning < 2.0`).

**Comment by BrunoPinos** [comment]
I have pytorch_lightning 1.9.5
![image](https://github.com/autogluon/autogluon/assets/58994038/69589934-a454-46e2-ad96-8b630c39cbf6)

pip doesn't signal any compatibility problem
![image](https://github.com/autogluon/autogluon/assets/58994038/87823072-6a85-4e52-aabd-cfdd3d0851de)

I'm on a virtual machine with linux

**Comment by BrunoPinos** [comment]
Ok I solved the problem !

I think I had the bug because I did:
pip install autogluon.timeseries ==0.8.1

instead of pip install autogluon ==0.8.1

Do you think it's possible to have acces to the DL model with autogluon.timeseries ?

**Comment by shchur** [comment]
If by "access to the model" you mean getting the model object, you can do that with
```python
# load model as autogluon.timeseries.models.AbstractTimeSeriesModel
model = predictor._trainer.load_model("DeepAR").most_recent_model
# Get GluonTS predictor
gts_predictor = model.gts_predictor
```

I'm not 100% sure about the second question, I recommend just trying and seeing if anything breaks.

**Comment by shchur** [comment]
Sorry for a delayed response. 

I was able to reproduce this problem locally:
- There is a bug in `tensorboard=2.5.0` that causes this issue. 
- An older version of `tensorboard` was present in the Python environment prior to installation of `autogluon.timeseries`, which causes this problem.  
- When a full version of autogluon is installed via `pip install autogluon`, tensorboard is updated to >=2.9 (thanks to requirements from `autogluon.multimodal`) and this problem is resolved.

I guess we should add a version cap on tensorboard to `autogluon.timeseries` to make sure that GPU training works in this setting.

Thanks a lot for reporting the problem @BrunoPinos!

**Comment by shchur** [comment]
Resolved by #3755
